wiki_articles/cp.md

```diff
@@ -1,38 +1,28 @@
-<!-- alias CP-help, CP -->
-
+<!-- alias cp-help, cp -->
 # The difficulty in helping with competitive coding problems
 
-Competitive programming problems are difficult to help with. The often require helpers to get very involved in problems, even taking the time to solve the problem themselves, and often also boil down to the person asking the question not having thought through the problem enough themselves to understand the complete.
-
-When asking a programming question, it's important to make the effort required for helpers as minimal as possible. 
-
-## How challenges are structured
-
-Competitive programming challenges often have arbitrary rules and requirements that require helpers to get deeply involved to even understand the problem.
-
-## Hidden test cases
+Please understand competitive programming (CP) questions are hard to help with:
```

Comment:
This sentence doesn't logically flow into the bullets. Something like `are hard to help with because:` would make sense.